docs/detections/api/exceptions/api-export-exception-list.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[exceptions-api-export-exception-list]]
+=== Export exception list
+
+Exports an exception list and its associated items to an `.ndjson` file.
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/exception_lists/_export`
+
+===== URL query parameters
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`id` |String |ID of the exception list. |No.
+|`list_id` |String |List ID of the exception list. |No.
```

Comment:
Both of these fields are required going back to when the route was created

Reply:
Resolved in [a56ddea](https://github.com/elastic/security-docs/pull/4316/commits/a56ddea5156c24f208b70a5d38570833073d288d).